package.json

```diff
@@ -16,7 +16,8 @@
     },
     "workspaces": [
         "packages/*",
-        "tools/*"
```

Comment:
this started breaking our packages - npm install packages based on directory order. This will also cause issues in the react-native demo, where react-native won't have access to packages needed to start the demo

I recommend to not do this now. We had this option but because of it we had a lot of troubles in the react/react-native project

Reply:
All right, closing for now.